### PR TITLE
Removed redundant spaces

### LINKED ISSUE #5892: Remove redundant spaces in multi-line strings
## Problem

Some multi-line strings contain redundant spaces 

https://github.com/mlflow/mlflow/blob/c635d1aa12e1749ab1321128ac61c0f3e6309c1d/examples/multistep_workflow/load_raw_data.py#L13-L16

They need to be fixed as follows:

```diff
diff --git a/examples/multistep_workflow/load_raw_data.py b/examples/multistep_workflow/load_raw_data.py
index 9d2ef4902..06c4e6864 100644
--- a/examples/multistep_workflow/load_raw_data.py
+++ b/examples/multistep_workflow/load_raw_data.py
@@ -12,7 +12,7 @@ import click
 
 @click.command(
     help="Downloads the MovieLens dataset and saves it as an mlflow artifact "
-    " called 'ratings-csv-dir'."
+    "called 'ratings-csv-dir'."
 )
 @click.option("--url", default="http://files.grouplens.org/datasets/movielens/ml-20m.zip")
 def load_raw_data(url):
```

## Instructions

1. Visit https://github.com/mlflow/mlflow
2. Press `.`.
3. Enable regular expression search.
4. Type the following in the search box. Note the second line ends with a space.
```
 "
\s+" 
```
5. Fix the lines found in the previous step.

image

### PR DESCRIPTION
**Pull request by PrajwalBorkar**
## Related Issues/PRs
Signed-off-by: PrajwalBorkar <prajwalborkar5075@gmail.com>

Closes #5892 

## What changes are proposed in this pull request?

Removed redundant spaces in multi-line strings.

## How is this patch tested?

Existing tests

## Does this PR change the documentation?

- [x] No. You can skip the rest of this section.
- [ ] Yes. Make sure the changed pages / sections render correctly by following the steps below.

1. Check the status of the `ci/circleci: build_doc` check. If it's successful, proceed to the
   next step, otherwise fix it.
2. Click `Details` on the right to open the job page of CircleCI.
3. Click the `Artifacts` tab.
4. Click `docs/build/html/index.html`.
5. Find the changed pages / sections and make sure they render correctly.

### Is this a user-facing change?

- [x] No. You can skip the rest of this section.
- [ ] Yes. Give a description of this change to be included in the release notes for MLflow users.

(Details in 1-2 sentences. You can just refer to another PR with a description if this PR is part of a larger change.)

### What component(s), interfaces, languages, and integrations does this PR affect?
Components
- [ ] `area/artifacts`: Artifact stores and artifact logging
- [ ] `area/build`: Build and test infrastructure for MLflow
- [ ] `area/docs`: MLflow documentation pages
- [ ] `area/examples`: Example code
- [ ] `area/model-registry`: Model Registry service, APIs, and the fluent client calls for Model Registry
- [ ] `area/models`: MLmodel format, model serialization/deserialization, flavors
- [ ] `area/projects`: MLproject format, project running backends
- [ ] `area/scoring`: MLflow Model server, model deployment tools, Spark UDFs
- [ ] `area/server-infra`: MLflow Tracking server backend
- [x] `area/tracking`: Tracking Service, tracking client APIs, autologging

Interface
- [ ] `area/uiux`: Front-end, user experience, plotting, JavaScript, JavaScript dev server
- [ ] `area/docker`: Docker use across MLflow's components, such as MLflow Projects and MLflow Models
- [ ] `area/sqlalchemy`: Use of SQLAlchemy in the Tracking Service or Model Registry
- [ ] `area/windows`: Windows support

Language
- [ ] `language/r`: R APIs and clients
- [ ] `language/java`: Java APIs and clients
- [ ] `language/new`: Proposals for new client languages

Integrations
- [ ] `integrations/azure`: Azure and Azure ML integrations
- [ ] `integrations/sagemaker`: SageMaker integrations
- [ ] `integrations/databricks`: Databricks integrations

<a name="release-note-category"></a>
### How should the PR be classified in the release notes? Choose one:

- [ ] `rn/breaking-change` - The PR will be mentioned in the "Breaking Changes" section
- [x] `rn/none` - No description will be included. The PR will be mentioned only by the PR number in the "Small Bugfixes and Documentation Updates" section
- [ ] `rn/feature` - A new user-facing feature worth mentioning in the release notes
- [ ] `rn/bug-fix` - A user-facing bug fix worth mentioning in the release notes
- [ ] `rn/documentation` - A user-facing documentation change worth mentioning in the release notes

### CODE REVIEW

File: mlflow/models/docker_utils.py [hunk 1]
```diff
@@ -98,11 +98,11 @@ def _get_mlflow_install_step(dockerfile_context_dir, mlflow_home):
         return (
             "RUN pip install mlflow=={version}\n"
             "RUN mvn "
-            " --batch-mode dependency:copy"
+            "--batch-mode dependency:copy"
```
Comment (harupy) [inline mlflow/models/docker_utils.py:101]: Can we revert this change and remove the space after `mvn`?

File: mlflow/models/docker_utils.py [hunk 2]
```diff
@@ -98,11 +98,11 @@ def _get_mlflow_install_step(dockerfile_context_dir, mlflow_home):
...
             " -Dartifact=org.mlflow:mlflow-scoring:{version}:pom"
             " -DoutputDirectory=/opt/java\n"
             "RUN mvn "
-            " --batch-mode dependency:copy"
+            "--batch-mode dependency:copy"
```
Comment (harupy) [inline mlflow/models/docker_utils.py:105]: same here: https://github.com/mlflow/mlflow/pull/5893/files#r875723728

File: mlflow/tracking/fluent.py [hunk 3]
```diff
@@ -131,7 +131,7 @@ def set_experiment(experiment_name: str = None, experiment_id: str = None) -> Ex
             message=(
                 "Cannot set a deleted experiment '%s' as the active experiment."
                 " You can restore the experiment, or permanently delete the "
-                " experiment to create a new one." % experiment.name
+                "experiment to create a new one." % experiment.name
```
Comment (harupy) [inline mlflow/tracking/fluent.py:134]: Can we remove the space before `You` in line 133 and add a space after `experiment.` in line 132?

**Comment by harupy** [comment]
@PrajwalBorkar Thanks for the PR, left some minor comments! LGTM once they are addressed :)

**Comment by harupy** [comment]
@PrajwalBorkar Can you fix the dco failure?

**Comment by harupy** [comment]
can you follow the instructions in https://github.com/mlflow/mlflow/pull/5893/checks?check_run_id=6489875096?

**Comment by PrajwalBorkar** [comment]
While adding signoff message in commit it says aborting commit due to empty message. I also tried -s flag but in vim it shows correct message but after committing it still shows error.

**Comment by harupy** [comment]
Didn't `git rebase HEAD~4 --signoff` work?

**Comment by harupy** [comment]
Ok, can you try this?

```
git reset $(git merge-base master $(git branch --show-current))
git add -A
git commit -s -m "Remove redundant spaces"
git push -f origin HEAD
```

**Comment by PrajwalBorkar** [comment]
> Didn't `git rebase HEAD~4 --signoff` work?

No maybe. Actually I tried all the commands given in 2 3 commits so I don't know exactly what is root of errors.

**Comment by PrajwalBorkar** [comment]
Screenshot (62)
Here is the log.

**Comment by harupy** [comment]
Correct

**Comment by harupy** [comment]
> Done. Any other changes required?

No :)

**Comment by PrajwalBorkar** [comment]
> > Done. Any other changes required?
> 
> No :)

Thanks a lot for the help. 
Also can u tell what I did wrong and what commands to run if issue appears again?

**Review by harupy — APPROVED** [review]
